src/substrait/textplan/converter/PipelineVisitor.cpp

```diff
@@ -121,18 +116,19 @@ std::any PipelineVisitor::visitRelation(
 
 std::any PipelineVisitor::visitPlanRelation(
     const ::substrait::proto::PlanRel& relation) {
-  auto* symbol = symbolTable_->lookupSymbolByLocation(PROTO_LOCATION(relation));
-  auto relationData = ANY_CAST(std::shared_ptr<RelationData>, symbol->blob);
+  auto symbols =
+      symbolTable_->lookupSymbolsByLocation(PROTO_LOCATION(relation));
+  auto relationData = ANY_CAST(std::shared_ptr<RelationData>, symbols[0]->blob);
```

Comment:
Do we maybe want a `lookupSingleSymbolByLocation` that does a check for us here that there is only one and gives an error otherwise?

Reply:
The symbol table class would need to return an error that would need to be reformatted here in the pipeline visitor.  In this particular case we know that every relation has to be in the symbol table as we visited all the relations in a previous visit to the parse tree.  Added a note.